tests/fakeapi/hello/__init__.py

```diff
@@ -87,8 +87,8 @@ def get_bye_secure_from_flask():
     return "Goodbye {user} (Secure!)".format(user=context["user"])
 
 
-def get_bye_secure_from_connexion(req_context):
-    return "Goodbye {user} (Secure!)".format(user=req_context["user"])
+def get_bye_secure_from_connexion():
+    return "Goodbye {user} (Secure!)".format(user=context["user"])
```

Comment:
We want to check that the context is passed to the function, so we shouldn't use the global context here.